### ISSUE TITLE
WebAIM Feedback - SC 1.4.13 – Printing

### ISSUE THREAD

**Issue by jared-w-smith** [w3c/wcag21#252]
The ability (or lack thereof) of user agents to properly apply user magnification to printed documents should not be the author’s responsibility. This success criterion falls well beyond the scope of WCAG. This SC would essentially require authors to become familiar with user agent print deficiencies and bugs, and force them to apply user agent specific `@media print` styles to address these.

WebAIM opposes the adoption of this SC as drafted.

**Comment by awkawk** [comment]
Thanks for the comment - the Working Group has not reached consensus on this proposed SC so it is deferred for future consideration.
